frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -53,16 +53,58 @@ const Controller: React.FC<ControllerProps> = ({
   render,
   application,
   cheURL,
+  namespace,
   resources,
   loaded,
   loadError,
   utils,
   serviceBinding,
   topologyFilters,
   trafficData,
-  helmResourcesMap,
-}) =>
-  render({
+}) => {
+  const secretCount = React.useRef<number>(0);
+  const helmResourcesMap = React.useRef<HelmReleaseResourcesMap>(null);
```

Comment:
This needs to be `useState` like before otherwise it won't trigger a re-rendering.